src/libraries/System.ComponentModel.TypeConverter/tests/TrimmingTests/System.ComponentModel.TypeConverter.TrimmingTests.proj

```diff
@@ -1,5 +1,20 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="ComObjectTypeTest.cs" />
+    <TestConsoleAppSourceFiles Include="DesigntimeLicenseContextSerialization_Serialize.cs" TrimmerRootAssemblies="mscorlib">
+      <ExtraTrimmerArgs>--feature System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization true</ExtraTrimmerArgs>
```

Comment:
Looks like these new tests need to be disabled on `browser-wasm`. `BinaryFormatter` isn't available on that platform.

You can do that like the following:

https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/System.Runtime/tests/TrimmingTests/System.Runtime.TrimmingTests.proj#L21-L25